src/Core/Framework/DataAbstractionLayer/Dbal/EntityWriteGateway.php

```diff
@@ -348,9 +350,19 @@ private function prefetch(EntityDefinition $definition, array $pks, WriteParamet
                     if ($id === null) {
                         continue 2;
                     }
+
                     $newIds[] = Uuid::fromHexToBytes($id);
                 }
 
+
```

Comment:
Sorry, this PR somehow slipped through the cracks 😬 So we can proceed, please remove this extra line break.